android/app/src/main/assets/index.android.bundle.meta

```diff
@@ -1 +1 @@
-�#4�,��Y��`�}��
\ No newline at end of file
+�*��0�ȃ_��;A��+�J�
```

Comment:
what is this file about?

Reply:
This is the Android's bundle information